roles/openshift_health_checker/openshift_checks/logging/elasticsearch.py

```diff
@@ -15,12 +15,10 @@ class Elasticsearch(LoggingCheck):
     name = "elasticsearch"
     tags = ["health", "logging"]
 
-    logging_namespace = None
-
     def run(self, tmp, task_vars):
         """Check various things and gather errors. Returns: result as hash"""
 
-        self.logging_namespace = get_var(task_vars, "openshift_logging_namespace", default="logging")
+        super(Elasticsearch, self).logging_namespace = get_var(task_vars, "openshift_logging_namespace", default="logging")
```

Comment:
```python
self.logging_namespace
```

`super()` can be a weird thing to understand -- https://docs.python.org/2/library/functions.html#super
Focus on the two moments when you need it:

1. Getting multiple inheritance right;
2. Accessing methods / attributes on the parent class without naming the parent class explicitly, typically when you are overriding a method with the same name (we do this a lot e.g. with `is_active`).

In all other cases, just let the language do the name look up in the standard way, `obj.method(...)` will look for `method` in `obj` and if not defined will recursively traverse up the class hierarchy until it finds `method` or eventually raise `AttributeError`.